pkg/cli/cliflags/flags.go

```diff
@@ -272,7 +272,7 @@ sessions.`,
 		Name:      "join",
 		Shorthand: "j",
 		Description: `
-The address of node which acts as bootstrap when a new node is
+The address of a node which acts as bootstrap when a new node is
```

Comment:
What about something similar to what we have in the docs now:

>The addresses for connecting a node to a cluster.
>
>When starting a multi-node cluster for the first time, set this flag to the addresses of 3-5 of the initial nodes. Then run the cockroach init command against any of the nodes to complete cluster startup.
>
>When starting a singe-node cluster, leave this flag out. This will cause the node to initialize a new single-node cluster without needing to run the cockroach init command. 
>
>When adding a node to an existing cluster, set this flag to 3-5 of the nodes already in the cluster; it's easiest to use the same list of addresses that was used to start the initial nodes.

Followed by the `This flag can be specified separately for each address...` bit.

Thoughts?